miden-tx/src/tests/mod.rs

```diff
@@ -414,8 +420,12 @@ fn test_empty_delta_nonce_update() {
     "
     );
 
-    let tx_script =
-        TransactionScript::compile(tx_script_src, [], TransactionKernel::assembler()).unwrap();
+    let tx_script = TransactionScript::compile(
+        tx_script_src,
+        [],
+        TransactionKernel::assembler_testing().clone(),
```

Comment:
Here, but also in all other similar places in this file, do we actually need to use the testing assembler? It seems like we are not importing `kernel` library anywhere and so, maybe the regular assembler will do?